clusters/clusters_api_sdk_test.go

```diff
@@ -0,0 +1,148 @@
+package clusters
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
These tests are the same as the ones in `clusters_api_test.go`, but with the `go-sdk`